docs/serving/samples/cloudevents/cloudevents-dotnet/Startup.cs

```diff
@@ -0,0 +1,42 @@
+using CloudNative.CloudEvents;
```

Comment:
We need license headers.

Reply:
Will add headers to all files. It seems like most of the samples we have don't have any license headers though, including the go sample I copied from :)

Reply:
Yeah, I figured 🙈